maud_macros/src/lib.rs

```diff
@@ -36,7 +36,7 @@ fn expand(input: TokenStream) -> TokenStream {
     let stmts = generate::generate(markups, output_ident.clone());
     quote!({
         extern crate maud;
-        let mut #output_ident = ::std::string::String::with_capacity(#size_hint);
+        let mut #output_ident = ::alloc::string::String::with_capacity(#size_hint);
```

Comment:
```
error[E0433]: failed to resolve: could not find `alloc` in the list of imported crates
  --> maud/tests/misc.rs:66:20
   |
66 |     let result_a = html! { (r) };
   |                    ^^^^^^^^^^^^^ not found in `alloc::string`
   |
```

You can probably fix this by adding `extern crate alloc;` above the existing `extern crate maud;` line.